### DISCUSSION TITLE
statistical data - yearly, monthly, weekly, daily

### DISCUSSION THREAD

**Discussion by mohan-unify-care** [mongodb-js/charts-embed-sdk#93] [Q&A] [answered]
I'm aiming to generate a chart and utilize it to view yearly, monthly, weekly, and daily statistics through the Embed SDK. Despite referencing several documents related to Atlas Charts, I couldn't find information on this. 

**Can someone explain if this is possible?**

**Comment by matt-d-rat** [comment]
@mohan-unify-care GitHub issues are for reporting bugs with the embedding SDK.

For general “how to” questions, please raise your question on the [discussions](https://github.com/mongodb-js/charts-embed-sdk/discussions) page, and we would be happy to answer (that way the whole user base benefits from the knowledge sharing).

**Comment by mohan-unify-care** [comment]
thank you for moving this to the discussion forum.
Can someone help on this please?

**Comment by kristinamongo** [comment, marked as answer]
Hi @mohan-unify-care 
Thanks for your interest in the Charts Embedding SDK.

Unfortunately at the moment you can't change the date binning options of a chart dynamically through the SDK.
This might be available at a later stage with the Charts API.

A workaround you can do now is to duplicate the chart and change the binning options so that you have a separate chart for yearly, monthly, weekly and daily statistics. You can allow them all for embedding, and you can render the specific chart from your app when you want to see a specific timeline.

I hope that helps. Let us know if you have any more questions.

**Reply by mohan-unify-care** [reply]
Thank you for your response. Eagerly anticipating the arrival of this feature.
